fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -366,76 +383,110 @@ private enum class Highlight(
+                        "Excellent craftsmanship",
+                    ),
+                ),
+            ),
+        )
+}
+
 @Composable
 @LightDarkPreview
-private fun ProductAnalysisPreview() {
+private fun ProductAnalysisPreview(
+    @PreviewParameter(ProductAnalysisPreviewModelParameterProvider::class) model: ProductAnalysisPreviewModel,
+) {
     FirefoxTheme {
         ReviewQualityCheckScaffold(
             onRequestDismiss = {},
         ) {
-            val productRecommendationsEnabled = remember { mutableStateOf(false) }
+            var productRecommendationsEnabled by remember { mutableStateOf(model.productRecommendationsEnabled) }
 
             ProductAnalysis(
-                productRecommendationsEnabled = productRecommendationsEnabled.value,
-                productAnalysis = AnalysisPresent(
-                    productId = "123",
-                    reviewGrade = ReviewQualityCheckState.Grade.B,
-                    analysisStatus = AnalysisStatus.UP_TO_DATE,
-                    adjustedRating = 3.6f,
-                    productUrl = "123",
-                    highlights = sortedMapOf(
-                        HighlightType.QUALITY to listOf(
-                            "High quality",
-                            "Excellent craftsmanship",
-                            "Superior materials",
-                        ),
-                        HighlightType.PRICE to listOf(
-                            "Affordable prices",
-                            "Great value for money",
-                            "Discounted offers",
-                        ),
-                        HighlightType.SHIPPING to listOf(
-                            "Fast and reliable shipping",
-                            "Free shipping options",
-                            "Express delivery",
-                        ),
-                        HighlightType.PACKAGING_AND_APPEARANCE to listOf(
-                            "Elegant packaging",
-                            "Attractive appearance",
-                            "Beautiful design",
-                        ),
-                        HighlightType.COMPETITIVENESS to listOf(
-                            "Competitive pricing",
-                            "Strong market presence",
-                            "Unbeatable deals",
-                        ),
-                    ),
-                ),
+                productRecommendationsEnabled = productRecommendationsEnabled,
+                productAnalysis = model.productAnalysis,
                 onOptOutClick = {},
                 onReanalyzeClick = {},
                 onProductRecommendationsEnabledStateChange = {
-                    productRecommendationsEnabled.value = it
+                    productRecommendationsEnabled = it
                 },
                 onReviewGradeLearnMoreClick = {},
                 onFooterLinkClick = {},
             )
         }
     }
 }
-
-@Composable
```

Comment:
Since the card previews are now visible in different `ProductAnalysisPreview`s we don't need this.